### PR TITLE
Add Baskervville

### PR DESCRIPTION
**Pull request by yanone**
She merged it ;)

### CODE REVIEW

**Comment by m4rc1e** [comment]
## Fontbakery report

Fontbakery version: 0.7.13

<details>
<summary><b>[5] Baskervville-Italic.ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Checking OS/2 achVendID.</summary>

* [com.google.fonts/check/vendor_id](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id)
* ⚠ **WARN** OS/2 VendorID value 'NONE' is not a known registered id. You should set it to your own 4 character code, and register that code with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx [code: unknown]

</details>
<details>
<summary>⚠ <b>WARN:</b> Stricter unitsPerEm criteria for Google Fonts. </summary>

* [com.google.fonts/check/unitsperem_strict](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/unitsperem_strict)
* ⚠ **WARN** Even though unitsPerEm (1000) in this font is reasonable. It is strongly advised to consider changing it to 2000, since it will likely improve the quality of Variable Fonts by avoiding excessive rounding of coordinates on interpolations. [code: legacy-value]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours.</summary>

* [com.google.fonts/check/contour_count](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/contour_count)
* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

Glyph name: yen	Contours detected: 3	Expected: 1 or 2
Glyph name: uni00B5	Contours detected: 0	Expected: 1
Glyph name: Eth	Contours detected: 3	Expected: 2
Glyph name: aogonek	Contours detected: 3	Expected: 2
Glyph name: Dcroat	Contours detected: 3	Expected: 2
Glyph name: dcroat	Contours detected: 3	Expected: 2
Glyph name: eogonek	Contours detected: 3	Expected: 2
Glyph name: hbar	Contours detected: 2	Expected: 1
Glyph name: uni2126	Contours detected: 0	Expected: 1
Glyph name: Lslash	Contours detected: 2	Expected: 1
Glyph name: lslash	Contours detected: 2	Expected: 1
Glyph name: OE	Contours detected: 3	Expected: 2
Glyph name: oe	Contours detected: 4	Expected: 3
Glyph name: Tbar	Contours detected: 2	Expected: 1
Glyph name: tbar	Contours detected: 2	Expected: 1
Glyph name: Uogonek	Contours detected: 2	Expected: 1
Glyph name: uogonek	Contours detected: 2	Expected: 1
Glyph name: daggerdbl	Contours detected: 2	Expected: 1 or 3
Glyph name: partialdiff	Contours detected: 0	Expected: 2
Glyph name: uni2206	Contours detected: 0	Expected: 2
Glyph name: product	Contours detected: 0	Expected: 1
Glyph name: summation	Contours detected: 0	Expected: 1
Glyph name: radical	Contours detected: 0	Expected: 1
Glyph name: integral	Contours detected: 0	Expected: 1
Glyph name: uni0394	Contours detected: 0	Expected: 2
Glyph name: uni03BC	Contours detected: 0	Expected: 1
Glyph name: pi	Contours detected: 0	Expected: 1
Glyph name: uni03A9	Contours detected: 0	Expected: 1 [code: contour-count]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
* ⚠ **WARN** This font lacks caret positioning values for these ligature glyphs:
	- f_f_h.liga
	- f_f_l.liga
	- f_j.liga
	- f_f_k.liga
	- f_f_b.liga
	- f_h.liga
	- t_t.liga
	- f_f_j.liga
	- fl

   [code: incomplete-caret-pos-data]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + b
	- b + f
	- f + h
	- h + f
	- f + i
	- i + f
	- f + j
	- j + f
	- f + k
	- k + f
	- f + l
	- l + b
	- h + i
	- i + j
	- j + k
	- k + l
	- l + t
	- t + t

   [code: lacks-kern-info]

</details>
<br>
</details>
<details>
<summary><b>[5] Baskervville-Regular.ttf</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Checking OS/2 achVendID.</summary>

* [com.google.fonts/check/vendor_id](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id)
* ⚠ **WARN** OS/2 VendorID value 'NONE' is not a known registered id. You should set it to your own 4 character code, and register that code with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx [code: unknown]

</details>
<details>
<summary>⚠ <b>WARN:</b> Stricter unitsPerEm criteria for Google Fonts. </summary>

* [com.google.fonts/check/unitsperem_strict](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/unitsperem_strict)
* ⚠ **WARN** Even though unitsPerEm (1000) in this font is reasonable. It is strongly advised to consider changing it to 2000, since it will likely improve the quality of Variable Fonts by avoiding excessive rounding of coordinates on interpolations. [code: legacy-value]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours.</summary>

* [com.google.fonts/check/contour_count](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/contour_count)
* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

Glyph name: yen	Contours detected: 3	Expected: 1 or 2
Glyph name: uni00B5	Contours detected: 0	Expected: 1
Glyph name: Eth	Contours detected: 3	Expected: 2
Glyph name: aogonek	Contours detected: 3	Expected: 2
Glyph name: Dcroat	Contours detected: 3	Expected: 2
Glyph name: dcroat	Contours detected: 3	Expected: 2
Glyph name: eogonek	Contours detected: 3	Expected: 2
Glyph name: hbar	Contours detected: 2	Expected: 1
Glyph name: uni2126	Contours detected: 0	Expected: 1
Glyph name: Lslash	Contours detected: 2	Expected: 1
Glyph name: lslash	Contours detected: 2	Expected: 1
Glyph name: oe	Contours detected: 4	Expected: 3
Glyph name: Tbar	Contours detected: 2	Expected: 1
Glyph name: tbar	Contours detected: 2	Expected: 1
Glyph name: Uogonek	Contours detected: 2	Expected: 1
Glyph name: uogonek	Contours detected: 2	Expected: 1
Glyph name: daggerdbl	Contours detected: 2	Expected: 1 or 3
Glyph name: partialdiff	Contours detected: 0	Expected: 2
Glyph name: uni2206	Contours detected: 0	Expected: 2
Glyph name: product	Contours detected: 0	Expected: 1
Glyph name: summation	Contours detected: 0	Expected: 1
Glyph name: radical	Contours detected: 0	Expected: 1
Glyph name: integral	Contours detected: 0	Expected: 1
Glyph name: uni0394	Contours detected: 0	Expected: 2
Glyph name: uni03BC	Contours detected: 0	Expected: 1
Glyph name: pi	Contours detected: 0	Expected: 1
Glyph name: uni03A9	Contours detected: 0	Expected: 1 [code: contour-count]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + b
	- b + f
	- f + h
	- h + f
	- f + i
	- i + f
	- f + j
	- j + f
	- f + k
	- k + f
	- f + l
	- l + b
	- h + i
	- i + j
	- j + k
	- k + l

   [code: lacks-kern-info]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 0 | 10 | 128 | 11 | 144 | 0 |
| 0% | 0% | 3% | 44% | 4% | 49% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**


## Diff images: [baskervville_qa.zip](http://35.188.158.120/media/5bf4b9fd-516f-4c1a-b33f-e972c42ba0e7/baskervville_qa.zip)

**Comment by m4rc1e** [comment]
@davelab6 This font is called "Baskervville". Will this cause us any problems?

**Comment by m4rc1e** [comment]
@yanone you're missing an OFL.txt license.

**Comment by yanone** [comment]
Oh, I didn't know it needed one. Added it. 
All my previous submissions here are missing it. Should be added to `gftools`, shouldn't it?

File: ofl/baskervville/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2011 by Anja Meiners, with Reserved Font Name 'ABeeZee'
```
Comment (m4rc1e) [inline ofl/baskervville/OFL.txt]: First line of each OFL.txt should be the same as the font's copyright string.

**Comment by m4rc1e** [comment]
Thanks @yanone.

**Comment by davelab6** [comment]
> @davelab6 This font is called "Baskervville". Will this cause us any problems?

I believe not.

**Pull request merged**